### PR TITLE
mod: update lndclient to use new SignOutputRaw

### PR DESCRIPTION
**Pull request by positiveblue**
We are not using taproot txs yet so we can just leave the new param to `nil`

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,23 +3,24 @@ module github.com/lightninglabs/pool
...
 	github.com/btcsuite/btcwallet/wtxmgr v1.5.0
 	github.com/davecgh/go-spew v1.1.1
 	github.com/decred/dcrd/dcrec/secp256k1/v4 v4.0.1
 	github.com/golang/mock v1.6.0
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.5.0
 	github.com/jessevdk/go-flags v1.4.0
-	github.com/lightninglabs/aperture v0.1.17-beta.0.20220325093943-42b9d4c1be7f
-	github.com/lightninglabs/lndclient v0.15.0-0
+	github.com/lightninglabs/aperture v0.1.17-beta.0.20220328072456-4a2632d0be38
+	github.com/lightninglabs/lndclient v1.0.1-0.20220328143152-e7bebec99366
```
Comment (guggero) [inline go.mod:20]: This has a tag now: `v0.15.0-1`

**Review by guggero — APPROVED** [review]
Nice, LGTM :tada:

**Review by bhandras — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (positiveblue) [reply go.mod:20]: I tried it before. I am not sure if I screwed up with the git tags, but if I use the `v0.15.0-1` I get

```
➜  pool git:(signoutputraw) go mod tidy      
go: finding module for package github.com/btcsuite/btcd/btcec
github.com/lightninglabs/pool imports
	github.com/lightninglabs/lndclient imports
	github.com/btcsuite/btcd/btcec: module github.com/btcsuite/btcd@latest found (v0.18.1), but does not contain package github.com/btcsuite/btcd/btcec
```

Comment (guggero) [reply go.mod:20]: Okay, there was something very weird with that tag... Created a new one (`v0.15.0-2`) and it seems to work now.

**Comment by dstadulis** [comment]
Next steps:
- merge in subasta
- adras needs to review
- rebase taproot PR on top of it

**Pull request merged**